mmhuman3d/utils/ffmpeg_utils.py

```diff
@@ -295,13 +295,14 @@ def video_to_array(
         width, height = int(info['width']), int(info['height'])
     num_frames = int(info['nb_frames'])
     start = (min(start, num_frames - 1) + num_frames) % num_frames
-    end = (min(end, num_frames - 1) + num_frames) % num_frames
+    end = (min(end, num_frames - 1) +
+           num_frames) % num_frames + 1 if end is not None else num_frames
```

Comment:
Similar as above. For both `start` and `end`.

Reply:
this is still useful since in `ffmpeg`, `start_frame` and `end_frame` must be None.